### PR TITLE
Adding GKE platform for user-agent

### LINKED ISSUE #68: Add FORSETI_PLATFORM env var
## Story
Add the FORSETI_PLATFORM (set to `gke`) env var into the [server pod configuration](https://github.com/forseti-security/helm-charts/blob/master/charts/forseti-security/templates/server/pod/_server-container.yaml).

### PR DESCRIPTION
**Pull request by kevensen**
Resolves #68

### CODE REVIEW

File: charts/forseti-security/templates/server/pod/_server-container.yaml [hunk 1]
```diff
@@ -49,6 +51,7 @@
       secretKeyRef:
         name: database-secret
         key: password
+>>>>>>> master
```
Comment (gkowalski-google) [inline charts/forseti-security/templates/server/pod/_server-container.yaml]: Artifact from a merge?

**Review by gkowalski-google — CHANGES_REQUESTED** [review]
Remove merge conflict line

File: charts/forseti-security/templates/server/pod/_server-container.yaml (hunk 1 above)
Comment (kevensen) [reply charts/forseti-security/templates/server/pod/_server-container.yaml]: yikes

**Review by gkowalski-google — APPROVED** [review]
LGTM, thanks!

**Pull request merged**